spec/support/vcr_cassettes/emis/get_military_service_episodes/valid_no_end_date_v2.yml

```diff
@@ -0,0 +1,72 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://localhost:4438/VIERSService/eMIS/v2/MilitaryInformationService
```

Comment:
The base url in this cassette is going to have to change to match the base in `settings.yml`: https://vaausvrsapp81.aac.va.gov